bin/fastatic

```diff
@@ -26,6 +27,7 @@ fastatic({
 		logger.log(`Done. Optimised static files in: ${dest}`);
 	})
 	.catch(err => {
-		console.error(err.message);
+		errorReporter(err, program.logLevel);
```

Comment:
If #98 is merged first, I agree `if (program.logLevel === 'verbose')` can be left out. But in that case the code at the top instantiating the logger still needs to change. If you add the `if (program.logLevel === 'verbose')` in this branch now, the other PR is not blocking it. Up to you.